test/bigint_test.dart

```diff
@@ -0,0 +1,36 @@
+//
+// bigint_test.dart
```

Comment:
This file actually has tests related to `LogicValue` operations between large `LogicValue`s.  Naming it `bigint_test` is maybe not quite right.  Maybe `big_logic_value_test`?  Or we could throw it in the (already huge) `logic_value_test.dart`.